Events/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Events
+layout: default
+author: Doug, Jackson, & Tony
+---
+More coming soon...
+================================
+
+# Upcoming Conferences
+* 17 January 2017 - American Nuclear Society: Oregon State University Chapter Winter Conference
```

Comment:
I'm getting nit picky now.... sorry.

~~University Chapter~~ University Student Section

it's a common misnomer. they're [student sections](http://www.ans.org/const/student/) not chapters.